dlrs/main/staticresources/DLRS_Logo.resource-meta.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<StaticResource xmlns="http://soap.sforce.com/2006/04/metadata">
+    <cacheControl>Private</cacheControl>
```

Comment:
We usually use public if nothing calls for private. Wonder what's your take on this?

Reply:
It just defaulted to private so I went with that, plus whenever I see "private" I tend to leave it that way unless we need to expose more access.

Though in this case since it's related to caching, it probably makes more sense to make it public. It's such a small thing that most users won't really see or be affected by so I guess no need to change it. But yeah, public for static resource caching seems like a good default going forward!